docs/migration.md

```diff
@@ -2,191 +2,188 @@
-        --restart=unless-stopped \
-        --privileged \
-        --network=host \
-        --volume /var/run/docker.sock:/var/run/docker.sock \
-    +   --volume ${your-data-dir}:/var/lib/gpustack \
-        --runtime nvidia \
-        gpustack/gpustack \
-        --server-url ${server-url} \
-        --token ${token}
-    ```
-
-    This will launch the GPUStack worker using your existing data and connect it to the specified server.
-
-- **Other GPU Architectures**
-
-    For architectures other than NVIDIA (e.g., AMD, Ascend), the migration process remains the same. To migrate on these platforms:
-
-    1. Get the installation commands, please refer to the commands in the [Installation Documentation](installation/requirements.md).
-
-    2. Mount the legacy data directory to `/var/lib/gpustack`.
-
-    3. (Server Only) Add the environment variable `GPUSTACK_MIGRATION_DATA_DIR` as shown in the NVIDIA examples, only the server needs to add this environment variable.
+!!! note
 
-    The Server and Worker migration commands can be used directly after applying these changes.
+    The support for GPU vendors has changed since version v2.0.0. Please refer to [Accelerator Runtime Requirements](./installation/requirements.md#accelerator-runtime-requirements).
 
-#### Recreating Model Instances
+## Migration Steps
 
-After the upgrade is complete, existing Model Instances may remain stuck in the `Starting` state. If this happens, recreating the Model Instance will allow the model to run normally.
+### Identify Your Legacy Data Directory
 
-#### Migration from llama-box
-If you were using llama-box as the inference backend in previous versions, please note that llama-box is no longer supported as of v2.0.0. Use llama.cpp via the custom inference backend instead.
+Locate the data directory used by your previous GPUStack installation. The default path is:
 
-1. Configure llama.cpp on Inference Backend page. For llama.cpp configuration, refer to this [document](./tutorials/using-custom-backends.md/#deploy-gguf-models-with-llamacpp).
-2. Go to the Deployment page, modify the model originally launched with llama-box, change the backend to llama.cpp
-3. Recreate the model instance after saving.
+```
+/var/lib/gpustack
+```
 
-!!! Note
+For other installation methods, refer to this [link](faq.md/#where-are-gpustacks-data-stored) to locate the data directory. In the following steps, this path is referenced as `${your-data-dir}`.
 
-    Distributed inference across multiple workers is currently not supported with custom inference backends.
+Due to architectural changes in v2.0.0, new components such as `postgres` and `s6-supervisor` may read from and write to files in `${your-data-dir}` using non-root users. Make sure that `${your-data-dir}` is accessible to all required users. You can use the following script to adjust permissions, but please be aware of the security risks and understand exactly what the script does before running it.
```

Comment:
Isn't it `777` that cause troubles?

Reply:
You don't need 777 for data-dir or it's parent. But it requires at least read privilege. for example:
- Lets make data-dir is `/mnt/a/b`
- For both `/mnt` and `/mnt/a`, they requires `a+x`, which means +0111 for their privilege.
- For both `/mnt/a/b` and `/mnt/a/b/log`, they requires `a+rx`, which means +0555 for it privilege.
- And the startup scripts for `postgres` and `-logger` components will create data dir/log dir which belongs to user `postgres` and user `nouser:nogroup`.

So in most case, it won't requires 777 for all directories.